src/sockperf.cpp

```diff
@@ -3038,13 +3024,20 @@ static int set_sockets_from_feedfile(const char *feedfile_name) {
         }
 #endif // NEED_REGEX_WORKAROUND
         else {
-            log_msg("Invalid input in line %s: "
-                    "each line must have the following format: ip:port or type:ip:port or "
-                    "type:ip:port:mc_src_ip. "
-                    "IPv6 addresses must be enclosed in square brackets.",
-                    line);
-            rc = SOCKPERF_ERR_INCORRECT;
-            break;
+            regexpres = regexec(&regexpr_unix, line, NUM_RE_GROUPS, m, 0);
+            if (regexpres == 0) {
+                type = line[0];
+                addr = std::string(line + m[1].rm_so, line + m[1].rm_eo);
+            } else {
+                log_msg("Invalid input in line %s: "
+                        "each line must have the following format: ip:port or type:ip:port or "
+                        "type:ip:port:mc_src_ip or type:/PATH. "
+                        "IPv6 addresses must be enclosed in square brackets."
+                        "Unix domain socket must start with '/'",
+                        line);
+                rc = SOCKPERF_ERR_INCORRECT;
+                break;
+            }
```

Comment:
Your code will not compile with something other than gcc 4.8.5.

You can reproduce compilation error on gcc 4.8.5 by commenting `#define NEED_REGEX_WORKAROUND` at the start of this file.

Reply:
not sure I understand. 
I saw the new `#define NEED_REGEX_WORKAROUND`
what is the issue?

Reply:
Check compilation logs ("Mellanox Lab" build below) to see how your code compiled using different compilers.

Reply:
Do we want to support new UDS feature with feedfile for old compiler?

Reply:
yes

Reply:
reminder